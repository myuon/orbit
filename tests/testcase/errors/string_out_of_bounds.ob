fun main() do
  let str = "Hi";
  return str[10];
end