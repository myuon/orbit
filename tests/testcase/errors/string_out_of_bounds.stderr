String index out of bounds